simplq/src/styles/buttons.module.scss

```diff
@@ -4,7 +4,7 @@
   background-color: $primary-color-dark;
   color: #fff;
   display: inline-block;
-  padding: 0.2em 0.5em 0.3em 0.5em;
+  padding: 0.2em 0.5em 0.3em 0.2em;
```

Comment:
Why did we reduce this padding?

Reply:
I thought it looked a little odd with the additional space to the left of the tick icon so I reduced it, but will revert it if you would like?

Reply:
I see, let's see how it looks.